overlord/snapstate/snapstate.go

```diff
@@ -285,3 +285,27 @@ func Set(s *state.State, name string, snapst *SnapState) {
 	snaps[name] = &raw
 	s.Set("snaps", snaps)
 }
+
+// All returns information about all active snaps.
+func All(s *state.State) ([]*snap.Info, error) {
```

Comment:
`All` => `ActiveInfos`

This is slightly different, but it's okay. We can have it for now anyway.